modules/nf-core/cellranger/mkgtf/main.nf

```diff
@@ -2,7 +2,7 @@ process CELLRANGER_MKGTF {
     tag "$gtf"
     label 'process_low'
 
-    container "nf-core/cellranger:9.0.1"
+    container "quay.io/nf-core/cellranger:9.0.1"
```

Comment:
no need to specify quay.io as it's the default registry for all nf-core

Reply:
But is it the same also when using it outside a nf-core pipeline? I did because it was going to dockerhub...

Reply:
all nf-core modules have this.

which is why we have this in the template:
https://github.com/nf-core/tools/blob/main/nf_core/pipeline-template/nextflow.config#L226-L233

Reply:
outside of nf-core I would recommend either using this default registry as well, or patching the module in the pipeline

Reply:
or overriding the container declaration in the config file